packages/ppx_deriving_encoding/ppx_deriving_encoding.0.3.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Ppx deriver for json-encoding"
+maintainer: "contact@origin-labs.com"
+authors: "Maxime Levillain <maxime.levillain@origin-labs.com"
+license: "LGPL-2.1-or-later"
+homepage: "https://gitlab.com/o-labs/ppx_deriving_encoding"
+bug-reports: "https://gitlab.com/o-labs/ppx_deriving_encoding/-/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "json-data-encoding" {>= "0.9"}
+  "ppxlib"
```

Comment:
you are missing lower-bound constraints here
```
# File "src/utils.ml", line 180, characters 38-47:
# 180 |   List.fold_left (fun fa a -> match a.attr_name.txt with
#                                             ^^^^^^^^^
# Error: Unbound record field attr_name
# Hint: Did you mean pmtd_name?
```

Reply:
yes I'm trying to put some ok constraint on it, I don't know why the CI goes as low as ocaml-migrate-parsetree.1.13, I think I will put ppxlib >= 0.16.0 and change a bit the code